docs/web/howto/web-in-60/websockets.rst

```diff
@@ -0,0 +1,91 @@
+:LastChangedDate: $LastChangedDate$
+:LastChangedRevision: $LastChangedRevision$
+:LastChangedBy: $LastChangedBy$
+
```

Comment:
Are we using these fields ?

I think that they were left from SVN times.

```suggestion
```

Reply:
I am not entirely sure of the purpose but I would rather just keep this file consistent with the other ones, and perhaps open a follow-up ticket to do a cleanup across the repo to remove these.